packages/block-editor/src/components/block-preview/style.scss

```diff
@@ -35,9 +35,15 @@
 	}
 }
 
-// Hide inserter from previews.
-.block-editor-block-preview__content-iframe .block-list-appender {
-	display: none;
+.block-editor-block-preview__content-iframe {
+	// Hide inserter from previews.
+	.block-list-appender {
+		display: none;
+	}
+
+	.block-editor-block-list__layout {
```

Comment:
Agree that, It would be easier test all Previews in a separate PR.

However, this style is to have margin inside of iframe (created by BlockPreview) and cannot be targeted from outside. And JS prop just for margin isn't a good idea. So, made this CSS part of this PR. Suggest if we should ship this change without a default spacing (preview below)

<img width="269" alt="image" src="https://user-images.githubusercontent.com/1935113/204252882-d5b76ff4-969b-4936-b589-fdf84bff9dc3.png">

Reply:
I think that's definitely something to be skipped from this PR. Also the `block-editor-block-list__layout` class is being added not only in the root container but also in every block that has inner blocks like `list, group` etc.. adding extra margins. This could be solved with a different selector, but still it affects all previews.

Personally I think without the margin this still is valuable addition.

Reply:
Removed the change from this PR. 👍